nncf/common/exporter.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         self._model = model
         self._input_names = input_names
         self._output_names = output_names
-        self._model_args = model_args if model_args else ({},)
+        self._model_args = model_args
```

Comment:
What is the reason for these changes?

Reply:
`None` should be the marker that the `model_args` were not provided, because it's easier to write a conditional checking that (compare `self._model_args is None` vs `self._model_args == ({},)`

Reply:
Ok, How did you check that this changes does not break something?

Reply:
I tend to rely on existing unit tests for that. In particular, `test_can_export_with_model_args` does that for PyTorch, and the TF doesn't use that arg at all. 

After you pointed this out, I actually discovered that I marked the corresponding test as xfail-ed at the same time when I added it. I removed the xfail and the test showed a bug in the implementation, which I fixed right now to make the test passing. So thanks for spotting this!